src/sentry/integrations/jira/integration.py

```diff
@@ -958,6 +961,48 @@ def get_resolve_sync_action(self, data: Mapping[str, Any]) -> ResolveSyncAction:
             should_unresolve=c_from in done_statuses and c_to not in done_statuses,
         )
 
+    def migrate_issues(self):
+        for project in Project.objects.filter(organization_id=self.organization_id):
+            plugin = None
+            for p in plugins.for_project(project):
+                if self.model.provider.startswith(p.slug) and p.get_option(
+                    "default_project", project
+                ):
+                    plugin = p
+                    break
+
+            if not plugin:
+                continue
+            if plugin.get_option("instance_url", project).rstrip("/") != self.model.metadata.get(
+                "base_url"
+            ).rstrip("/"):
+                continue
+            migrate_issues.apply_async(
+                kwargs={
+                    "integration_id": self.model.id,
+                    "organization_id": self.organization_id,
+                    "project_id": project.id,
+                    "plugin_slug": plugin.slug,
+                }
+            )
+
+            plugin_ignored_fields = plugin.get_option("ignored_fields", project)
+            if plugin_ignored_fields:
+                logging_context = {
+                    "organization_id": self.organization_id,
+                    "project_id": project.id,
+                    "plugin_slug": plugin.slug,
+                }
+                migrate_ignored_fields.apply_async(
+                    kwargs={
+                        "integration_id": self.model.id,
+                        "plugin_ignored_fields": plugin_ignored_fields,
+                        "logging_context": logging_context,
+                    }
+                )
+
+            plugin.disable(project)
```

Comment:
Once everything is migrated, the plugin is disabled. I do want to call out that we've decided not to migrate the `auto_create` option - if they had it set to True then it would create a Jira ticket for every new Sentry issue that came into the project. When the plugin is disabled, this functionality will cease. 

Users on the Business plan can manually set up alert rules to recreate this behavior (and probably will set more restrictive settings than whenever an issue comes in as that's quite noisy), but the amount of work it'd take to build this out separately for Jira and Jira Server is medium sized as we'd need to ensure the Jira/Server integration has access to the project and issue type, and that all the metadata fields that are set in the plugin are valid for the integration, and then create an alert rule out of that information.

Reply:
Do you think it's worth calling out that this behaviour will change for some customers?

Reply:
Where would I call it out? I made the front end ticket for this which should communicate as much.